docs/simulate/pauli_observables.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>maybe note that this may not be the most efficient way to do this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Added.